run-checks

```diff
@@ -216,7 +216,7 @@ if [ "$STATIC" = 1 ]; then
                 find . \( -name .git -o -name vendor \) -prune -o -print0 ) |
             xargs -0 file -N |
             awk -F": " '$2~/shell.script/{print $1}' |
-            xargs shellcheck
+            xargs shellcheck -S warning
```

Comment:
This looks risky, the error indicates that "shellcheck -x" will make the problem go away. But I also wonder why I don't see these warnings when I run a local ./run-checks --static - did something change between 18.04 and 20.04 here?

Reply:
I agree, we wat shellcheck -x.

Reply:
I was puzzled why we get different output here compared to the when we run the shellcheck tests inside gihub action as a static check. The differences are:
- find above is used instead of git ls-files because this runs not in a git checkout in the unit test
- the shellcheck deb is used instead of the shellcheck snap but stable should be the same version

I'm still puzzled why we see the warnings here but not in gh actions.